### DISCUSSION TITLE
未找到相关的 Issues 进行评论，问题出在哪里

### DISCUSSION THREAD

**Discussion by zkrisj** [gitalk/gitalk#526] [Q&A]
```
var gitalk = new Gitalk({
  clientID: '8ca2c4dd3df9c5c52775',
  clientSecret: 'c8dd6b595f51083ed79f21b241f2abff46dff2ec',
  repo: 'zkrisj.github.io',
  owner: 'zkrisj',
  admin: ['zkrisj'],
  id: location.pathname,
});
```
下面是 OAuth application settings 页面配置
Application name
zkrisj
Homepage URL
https://zkrisj.github.io
Authorization callback URL
https://zkrisj.github.io

网络请求返回空数组
https://api.github.com/repos/zkrisj/zkrisj.github.io/issues?labels=Gitalk,%2Fjs_tutorial%2F1.html&t=1661701740207

**Comment by balloonwj** [comment]
我和你问题一样，请问下，后来你解决了吗？

**Reply by balloonwj** [reply]
这个我后来解决了，其实就是授权问题。有时候网络不好，第一次在评论框登录github给gitalk授权仓库读写权限时没成功，建议清理浏览器缓存，再授权一次， 就解决了。

**Comment by ghost** [comment]
救命 我也是
